google_guest_agent/addresses.go

```diff
@@ -532,8 +532,22 @@ func enableNetworkInterfaces() error {
 }
 
 func isNetworkManagerRunning() bool {
-	res := runCmd(exec.Command("systemctl", "is-active", "--quiet", "NetworkManager"))
-	return res == nil
+	// SystemD
+	if systemctl, err := exec.LookPath("systemctl"); err == nil {
+		return runCmd(exec.Command(systemctl, "is-active", "--quiet", "NetworkManager")) == nil
```

Comment:
this assumes that if systemctl exists, that systemd is running. that's usually true but not necessarily. might want to switch this to checking for the existence of /var/run/systemd, which means systemd is running. or just look for running network manager process.